java-frontend/src/main/java/org/sonar/java/resolve/TypeVariableJavaType.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.sonar.java.resolve;
 
+import java.util.ArrayList;
 import org.sonar.plugins.java.api.semantic.Type;
 
 import java.util.List;
 
 public class TypeVariableJavaType extends JavaType {
 
-  List<JavaType> bounds;
+  List<JavaType> bounds = new ArrayList<>();
```

Comment:
Why do you need this? It seems to me that bounds are ALWAYS set beforehand. Isn't it going to make the computation of erasure fail in some case?! (`bounds.get(0)`)